src/Mount.cpp

```diff
@@ -473,12 +179,12 @@ void Mount::writePersistentData(int which, long val)
 //
 /////////////////////////////////
 #if RA_STEPPER_TYPE == STEPPER_TYPE_28BYJ48
-void Mount::configureRAStepper(byte stepMode, byte pin1, byte pin2, byte pin3, byte pin4, int maxSpeed, int maxAcceleration)
+void Mount::configureRAStepper(byte pin1, byte pin2, byte pin3, byte pin4, int maxSpeed, int maxAcceleration)
 {
 #if NORTHERN_HEMISPHERE
-  _stepperRA = new AccelStepper(stepMode, pin4, pin3, pin2, pin1);
+  _stepperRA = new AccelStepper((RA_SLEW_MICROSTEPPING == 1) ? AccelStepper::FULL4WIRE : AccelStepper::HALF4WIRE, pin4, pin3, pin2, pin1);
 #else
-  _stepperRA = new AccelStepper(stepMode, pin1, pin2, pin3, pin4);
+  _stepperRA = new AccelStepper((RA_SLEW_MICROSTEPPING == 1) ? AccelStepper::FULL4WIRE : AccelStepper::HALF4WIRE, pin1, pin2, pin3, pin4);
```

Comment:
mh ... luckily all this will change with HAL anyway :)

Reply:
Yeah. I moved it in from setup to make it clearer - one step closer to the HAL. I am trying to do these changes incrementally - hopefully they will accumulate into natural groupings that can be easily copied to the HAL implementation, otherwise we will be changing everything everywhere and merging will be a nightmare.